msteams-platform/m365-apps/cards-loop-component.md

```diff
@@ -61,27 +64,31 @@ The following is a JSON example of an Adaptive Card-based Loop component with th
 
 ### Query parameters
 
-|Property|Type|Description|Required|
-|---|---|---|---|
-| `metadata`| Metadata | Defines various metadata properties typically not used for rendering the card. | No |
-| `webUrl` | String | URL that uniquely identifies the card and serves as a browser fallback that can be used by some hosts.| No |
+|Property|Type|Description|
+|---|---|---|
+| `metadata`| Metadata | Defines various metadata properties typically not used for rendering the card. |
+| `webUrl` | String | URL that uniquely identifies the card and serves as a browser fallback that can be used by some hosts. |
+
+> [!NOTE]
+> * When you copy an Adaptive Card-based Loop component, the URL of the Loop component is copied. When you paste the URL in a Teams chat or the Loop app, the link unfurls into the Adaptive Card-based Loop component.
+> * Adaptive Card-based Loop components use the same [refresh mechanism](/adaptive-cards/authoring-cards/universal-action-model#refresh-mechanism) as regular Adaptive Cards. The content in an Adaptive Card-based Loop component is refreshed only when a user opens the Loop component.
 
-## Test your Loop component
+## Test Adaptive Card-based Loop component
 
-You can test the Loop component in the developer environments of Teams and Outlook for web.
+You can test Adaptive Card-based Loop components in Teams and Outlook.
 
-### Test in Microsoft Teams
+### Test in Teams
 
-To configure, distribute, and manage your application use the [Developer Portal for Teams](../concepts/build-and-test/teams-developer-portal.md). You can test and debug your app in the Developer Portal using the following options:
+You can test and debug your app in [Developer Portal for Teams](../concepts/build-and-test/teams-developer-portal.md) using the following options:
 
-* **Overview page**: On the **Overview page**, under **Teams store validation**, you can see a snapshot of your app's configuration and check if your app package validates against Microsoft Teams Store test cases.
+* **Overview page**: On the **Overview page**, under **Teams Store validation**, you can see a snapshot of your app's configuration and check if your app package validates against Microsoft Teams Store test cases.
 * **Preview in Teams**: The **Preview in Teams** button launches your app quickly in the Teams client for debugging.
 
 :::image type="content" source="../assets/images/developer-portal-overview.png" alt-text="A screenshot of the Developer Portal overview page with the Preview in Teams button highlighted. " lightbox="../assets/images/developer-portal-overview.png" :::
 
-### Test in Outlook for web
+### Test in Outlook
 
-To turn on the Adaptive Card-based Loop component in Outlook for web, follow these steps:
+To turn on the Adaptive Card-based Loop component in Outlook, follow these steps:
 
 1. Create a search-based message extension using [Teams App Camp](https://microsoft.github.io/app-camp/).
```

Comment:
This seems an odd requirement for testing a AC Loop component ME, since this point in the tutorial assumes the reader already has one. If we want to retain this step, maybe we should at least point to the sample app for this article: https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-unfurling-ac-loop-components/nodejs

Reply:
I've removed this section.